tests/templates/generation_test.py

```diff
@@ -0,0 +1,32 @@
+from client.utils.auth import success_page, success_auth, success_courses, get_file
+import unittest
+
+class SuccessPageTest(unittest.TestCase):
+	
+	def write(self, file, html):
+		get_file('../../tests/templates/'+file, 'w').write(html)
+	
+	def testCourseGeneration(self):
+		response = '[{"year": "2015", "institution": "UC Soumya", "url": "https://ok-server.appspot.com/#/course/5066549580791808", "display_name": "CS 61A", "term": "spring"}]'
+		html = success_courses(response)
+		self.write('test.partial.course.html', html)
+		assert html.find('2015') != -1
```

Comment:
Please use `self.assertNotEqual(-1, html.find('2015'))`; the error message is more descriptive.